medium/material_icons/dartpad_metadata.yaml

```diff
@@ -0,0 +1,4 @@
+name: Material Icons Demonstration
+mode: flutter
+files:
+  - name: main.dart
```

Comment:
Maybe add a newline? Git shows "no newline at end of file"

Reply:
Done